src/KeyVault/KeyVault/Commands/FullBackupRestore/FullBackupRestoreCmdletBase.cs

```diff
@@ -64,7 +69,12 @@ private void PreprocessParameterSet()
 
             if (this.IsParameterBound(c => c.StorageAccountName))
             {
-                StorageContainerUri = new Uri($"https://{StorageAccountName}.{DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix)}/{StorageContainerName}");
+                StorageContainerUri = new Uri($"https://{StorageAccountName}.blob.{DefaultContext.Environment.GetEndpoint(AzureEnvironment.Endpoint.StorageEndpointSuffix)}/{StorageContainerName}");
+            }
+
+            if (this.IsParameterBound(c => c.SasToken) && this.IsParameterBound(c => c.UseUserManagedIdentity))
+            {
+                throw new AzPSArgumentException("Parameter SasToken and UseUserManagedIdentity can not exist at the same time. Please choose either one as authentication method.", ErrorKind.UserError);
```

Comment:
If customer provides sastoken and UseUserManagedIdentity at the same time, I'd suggest using sastoken by default to follow the design of [swagger](https://github.com/Azure/azure-rest-api-specs/blob/05c4049bc22f0ec65acc18f9835132397049cb9e/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json#L356).

Needs to check if sastoken and UseUserManagedIdentity both are not provided.